### DISCUSSION TITLE
User Type and User Id are NULL, any idea why?

### DISCUSSION THREAD

**Discussion by alexfleites** [owen-it/laravel-auditing#949] [Q&A]
For some models user_type and user_id are NULL, and even when the models has the same configuration as others.

**Comment by parallels999** [comment]
Check user resolver
https://github.com/owen-it/laravel-auditing/blob/master/src/Resolvers/UserResolver.php
Make a debug

Maybe related: https://github.com/owen-it/laravel-auditing/issues/918

**Comment by alexfleites** [comment]
Yes, was the first thing I did, and noticed that Auth::guard($guard)->user() always is null, then added the middleware ['auth:api'] to the route and now return 401 Unauthenticated... any help would be appreciated.

**Reply by erikn69** [reply]
Sure, make a repo with a fresh laravel installation showing the problem
